app/javascript/shared/components/Radio.css

```diff
@@ -36,16 +36,10 @@
   transform: scale(0);
 }
 
-.radio-label:hover span:first-child,
-.radio-input:focus + label span:first-child {
+.radio-label:hover span:first-child {
   @apply bg-primary-100 border-primary-500;
 }
 
-.radio-input {
-  top: 6px;
-  left: 6px;
-}
-
```

Comment:
@indicozy - This file has changes unrelated to this PR. Could you please stash or move them to a new PR?